documentation/slate/source/includes/apis.md

```diff
@@ -411,9 +410,9 @@ Response example for :
 
 Also known as `/places` service.
 
-This api search in all geographical objects via their names. It returns,
-in addition of classic objects, a collection of [places](#place) . This api is
-very useful to make some autocomplete stuff.
+This endpoint allows you to search in all geographical objects using their names, returning
+a [places](#place) collection.
+It is very useful to make some autocomplete stuff.
```

Comment:
Are you trying to say that it is usefull to help understand what place the user is looking for, even if he has mittens? ;)
-> This sentence is not so clear to me.